### DISCUSSION TITLE
Why i can't set "null" as default value for nullable columns?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [prisma/prisma#8644] [Q&A] [answered]
```
model Comment {
  id           Int                @id @default(autoincrement())
  reply      Comment?   @relation(fields: [replyId], references: [id], onDelete: SetNull)
  replyId   Int?              @default(null)
  comment   String
}
```
Here both reply and replyId are nullable, But when I want to migrate I get this error:
```
error: Error parsing attribute "@default": Expected a numeric value, but received literal value "Null".
  -->  schema.prisma:70
   |
69 |   reply     Comment? @relation(fields: [replyId], references: [id], onDelete: SetNull)
70 |   replyId   Int?     @default(null)
   |
```
I'm new to Prisma. Can anyone explain the problem?

**Comment by ryands17** [comment, marked as answer]
@ahhshm 👋 

Databases already default to `null` if the value is optional. So in a Prisma model, when you define an optional field like this:

```prisma
replyId Int?
```

You don't need to specify `null` as a default as the record will automatically be `null` if nothing you don't pass any value to it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
